fdbserver/fdbserver.actor.cpp

```diff
@@ -1142,6 +1148,15 @@ struct CLIOptions {
 				localities.set(key, Standalone<StringRef>(std::string(args.OptionArg())));
 				break;
 			}
+			case OPT_IP_TRUSTED_MASK: {
+				Optional<std::string> subnetKey = extractPrefixedArgument("--trusted-subnet", args.OptionSyntax());
+				if (!subnetKey.present()) {
+					fprintf(stderr, "ERROR: unable to parse locality key '%s'\n", args.OptionSyntax());
+					flushAndExit(FDB_EXIT_ERROR);
+				}
+				allowList.addTrustedSubnet(args.OptionArg());
```

Comment:
You can add as many as you want. This command line argument works similar to the `locality` argument as it will take a key and a value (the key will be mostly there for documentation purposes, so we don't use it yet in FDB). For example you can do something like this:

```
bin/fdbserver --trusted-subnet-xp 10.0.0.1/24 --trusted-subnet-gs 192.168.0.1/24
```